### ISSUE TITLE
Democrat training tweets tested on Republican tweets => Getting 100% accuracy

### ISSUE THREAD

**Issue by BSkedgell** [RandallPark/Language-Processing-Project#2]
Hey @fireproofsocks ,

Working on our natural language processing of tweets from both political parties in this repository under the ipynb named "democrat_train_with_republican_test_nlp-ml_bskedgell" which is based off of the ipynb named "Practice_NLP" in this directory (assuming were inside the NLP folder of the Language-Processing-Project).

In the example they have some IMBD reviews they are trying to predict and are able to get the ML portion of the ipynb to achieve an accuracy of 86.56%.

When we feed our version of the ML with our tweet data processed by the NLP, first with Democrat tweets to train, then Republican tweets to test. I would expect to get a very low accuracy rating if the model was working correctly (should not be accurate if every prediction it makes is democrat is really a republican in test) but instead we are getting back 100% accuracy.

Was wondering if you could take a look at the notebook and see if there are any issues you can see easily and then we can try to address. The notebook should run from top to bottom no problem. 

Tutorial number 20 is what we are using as practice in the ipynb named "Practice_NLP" the link to the tensorflow tutorial is here:
https://www.tensorflow.org/tutorials/keras/basic_text_classification
